hazelcast-integration/operator-framework/README.md

```diff
@@ -0,0 +1,168 @@
+# Hazelcast Operator
+
+Hazelcast Enterprise is packaged with [Operator Framework](https://github.com/operator-framework), which simplifies deployment on Kubernetes and OpenShift. This is a step-by-step guide how to deploy Hazelcast cluster together with Management Center on your Kubernetes or OpenShift cluster.
+
+## Prerequisites
+
+You must have one of the followings:
+ * Kubernetes cluster (with admin rights) and the `kubectl` command configured (you may use [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/))
+ * OpenShift cluster (with admin rights) and the `oc` command configured (you may use [Minishift](https://github.com/minishift/minishift))
+
+## Kubernetes Deployment steps
+
+Below are the steps to start a Hazelcast Enterprise cluster using Operator Framework. Note that the first 3 steps are usually performed only once for the Kubernetes cluster (by the cluster admin). The step 4 is performed each time you want to create a new Hazelcast cluster.
+
+#### Step 1: Create RBAC
+
+Run the following commands to configure the Operator permissions.
+
+    kubectl apply -f rbac.yaml
+
+#### Step 2: Create CRD (Custom Resource Definition)
+
+To create the Hazelcast resource definition, run the following command.
+
+    kubectl apply -f hazelcastcluster.crd.yaml
+
+#### Step 3: Deploy Hazelcast Operator
+
+Deploy Hazelcast Enterprise Operator with the following command.
+
+    kubectl apply -f operator.yaml
+
+#### Step 4: Start Hazelcast
+
+Add a Secret within the Project that contains the Hazelcast License Key. If you don't have one, get a trial key from this [link](https://hazelcast.com/hazelcast-enterprise-download/trial/).
+
+    kubectl create secret generic hz-license-key-secret --from-literal=key=LICENSE-KEY-HERE
+
+Then, start Hazelcast cluster with the following command.
+
+    kubectl apply -f hazelcast.yaml
+
+Your Hazelcast Enterprise cluster (together with Management Center) should be created.
+
+    $ kubectl get all
+    NAME                                                                  READY   STATUS    RESTARTS   AGE
+    pod/hazelcast-operator-79468c667-lz96b                                1/1     Running   0          6m
+    pod/hz-9hwgko3mhi7u5esbwrelk0kj0-hazelcast-enterprise-0               1/1     Running   0          1m
```

Comment:
I would mention the name size limitation with the pod name. Some might hit that problem.

Reply:
Good point, I'll add it to the troubleshooting section.